File: app/src/main/java/com/selfmate/mes/selfmate/STATIC_RESOURCES/TOPIC.java
package com.selfmate.mes.selfmate.STATIC_RESOURCES;

/**
 * Created by vicki_mes on 11/20/2017.
 */

public class TOPIC {
    public static int MOVIE =1;
    public static int FINANCE =2;
    public static int TECHNOLOGY =3;
    public static int FOOD =4;
    public static int SPORT =5;
    public static int SCIENCE =6;
    public static int BRANDS =7;
    public static int HOT =8;

    public  static String getTopic(int i){
        switch (i){
            case 1:
                return "MOVIE";

            case 2:
                return "FINANCE";

            case 3:
                return "TECHNOLOGY";
            case 4:
                return "FOOD";
            case 5:
                return "SPORT";
            case 6:
                return "SCIENCE";
            case 7:
                return "BRANDS";
            case 8:
                return "HOT";
            default:return "N.O.N";
        }
    }
}

File: app/src/main/java/com/selfmate/mes/selfmate/si/TOPIC.kt
package com.selfmate.mes.selfmate.si

/**
 * Created by vicki_mes on 11/20/2017.
 */

object TOPIC {
    val BRANDS = 1
    val FINANCE = 2
    val FOOD = 3
    val MOVIE = 4
    val SCIENCE = 5
    val SPORT = 6
    val TECHNOLOGY = 7
    val HOT = 8

    fun getTopic(i: Int): String = when (i) {
        1 -> "BRANDS"
        2 -> "FINANCE"
        3 -> "FOOD"
        4 -> "MOVIE"
        5 -> "SCIENCE"
        6 -> "SPORT"
        7 -> "TECHNOLOGY"
        8 -> "HOT"
        else -> "N.O.N"
    }
}
